test/defineVariable.test.js

```diff
@@ -0,0 +1,64 @@
+const fhirpath = require("../src/fhirpath");
+const r4_model = require("../fhir-context/r4");
+const _ = require("lodash");
+const input = {
+  get patientExample() {
+    // Clone input file contents to avoid one test affecting another
+    return _.cloneDeep(require("../test/resources/patient-example.json"));
+  },
+};
+
+describe("defineVariable", () => {
+
+  it("simple use of a variable", () => {
+    let expr = `defineVariable('fam', name.first()).select(%fam.given)`;
```

Comment:
If you aren't embedding variables in these template literals, could you switch to double-quotes?

Reply:
udpated - only 1 test remains with the backtic due to it being a multi-line string.